main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
+- Come chat with us and other developers on the Official [Agoric Discord](https://agoric.com/discord)
+- Search and post [Q & A](https://github.com/Agoric/agoric-sdk/discussions/categories/q-a) in [agoric-sdk discussions](https://github.com/Agoric/agoric-sdk/discussions)
+- Visit and Tag [Agoric on X](https://twitter.com/agoric)
+- Send an Email to [Developer Relations](mailto://kbennett@agoric.com)
 
-::: tip Beta Dapp Architecture
-The Dapp architecture presented here is a beta preview of
-our eventual permissionless contract deployment model,
-extending our [Distributed Computing Framework](../js-programming/)
-to a stateful peer on end-user machines.
 
-The [Mainnet-1 launch](https://agoric.com/blog/announcements/agoric-composable-smart-contract-framework-reaches-mainnet-1-milestone)
-uses a ["smart wallet" architecture](./contract-rpc.md) with a lower client-side footprint.
-:::
+# Welcome!
+In this tutorial you will install the Agoric SDK as well as a simple application to test the functionality of Agoric.
+Currently Agoric supports [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about), MacOS, and Linux. This tutorial is based on an installation of [Ubuntu 22.04 LTS](https://ubuntu.com/download/desktop). If you're using a different operating system, some variation may be required.
 
-We'll be running **three terminal windows**. See below: 
 
- 1. ```sh
-    # Terminal 1: simulated blockchain and "solo" client
-    ```
- 2. ```sh secondary style2
-    # Terminal 2: contract interaction
-    ```
- 3. ```sh secondary style3
-    # Terminal 3: web user interface
-    ```
+# Installing Curl
+Begin by installing the `curl` utility, if it's not already installed. If using Ubuntu you'll first need to run the command below to prevent an error when installing `curl`.
+`sudo apt-get update --fix-missing`
 
-::: tip Watch: Prepare Your Agoric Environment (November 2020)
-This presentation includes starting a project, but note an outdated detail:
+![006](./assets/006.png)
 
- - In the REPL `x~.go()` tildot support has been postponed; use `E(x).go()` instead.
+Install the `curl` utility.
+`sudo apt install curl`
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/w0By22jYhJA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-:::
+![007](./assets/007.png)
 
-## Initialize the Default Dapp Template
 
-The following section will explain how to initialize the default Dapp template, and install the Agoric SDK
-into the Dapp template.
+# Installing NVM and Node v18.16.0
+At this point the Node Version Manager (NVM) utility will be installed. NVM makes it easy to select the specific version of Node that will be required (v18.16.0).
+`curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash`
```

Comment:
This is likely someone's first contact with Agoric, and I believe first impressions really matter. I want success to be the outcome 100% of the time someone runs through these steps, or else we risk losing that new dev. So, that's why I'm a little hesitant to just assume they can use a browser to do this, or complete any steps that aren't explicitly documented. I'm open to a better, more secure and brand-consistent way to do this, I just used these steps because it's what they have in their official docs: https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating